src/Ledger/Fees.lagda

```diff
@@ -69,15 +68,13 @@ scriptsCost pp scSz with (PParams.refScriptCostStride pp)
 \caption{Calculation of fees for reference scripts}
 \label{fig:scriptsCost}
 \end{figure*}
-
 The function \scriptsCost{} (Fig.~\ref{fig:scriptsCost}) calculates
 the fee for reference scripts in the transaction using a function that
 is piece-wise linear in the size, where the linear constant multiple
 grows with each \refScriptCostStride{} bytes. Thus, the \scriptsCost{}
 function depends on the \AgdaFunction{scriptsTotalSize} function,
 which returns an integer that is the total size of the reference
 script in bytes, as well as the following protocol parameters:
-%
```

Comment:
I added '%' between the paragraph and itemize since the latter is part of the former.

Reply:
Okay, I just deleted the line altogether (since they are closely associated).  I only did that because I needed to make a trivial change so the make docs command would recompile instead of complaining that everything is up to date.  (And I only needed to do that because of the tex error about `l'.)